Lib/ctypes/__init__.py

```diff
@@ -563,4 +564,9 @@ def DllCanUnloadNow():
     elif sizeof(kind) == 8: c_uint64 = kind
 del(kind)
 
+if SIZEOF_TIME_T == 8:
+    c_time_t = c_int64
+elif SIZEOF_TIME_T == 4:
+    c_time_t = c_int32
+
```

Comment:
@thp 
Is it ok not to define else case?

Reply:
@thp  
I think you should add to raise an exception on else case to make sure.
It may not happen, but it is a good practice to consider such exceptional case.

Reply:
PR updated, now raises an exception.

Reply:
It seems I had approved this PR yesterday before the PR was updated,
But it was a mistake. I made it on a smart phone.

It seem I can't dismiss  the approval for this repository.